freqtrade/freqtradebot.py

```diff
@@ -888,24 +906,44 @@ def check_handle_timedout(self) -> None:
                                            default_retval=False)(pair=trade.pair,
                                                                  trade=trade,
                                                                  order=order))):
-                reason = self.handle_timedout_limit_sell(trade, order)
-                self.wallets.update()
-                order_type = self.strategy.order_types['sell']
-                self._notify_sell_cancel(trade, order_type, reason)
+                self.handle_cancel_sell(trade, order, CancelReason.TIMEOUT.value)
 
-    def handle_timedout_limit_buy(self, trade: Trade, order: Dict) -> bool:
+    def cancel_all_open_orders(self) -> None:
         """
-        Buy timeout - cancel order
+        Cancel all orders that are currently open
+        :return: None
+        """
+
+        for trade in Trade.get_open_order_trades():
+            try:
+                if not trade.open_order_id:
+                    continue
```

Comment:
I don't think this is necessary.
`Trade.get_open_order_trades()` does the query via `Trade.open_order_id.isnot(None)` - so i don't think it's possible to hit this "continue" statement.